Tests/BotTests/MergeServiceHealthcheckTests.swift

```diff
@@ -8,33 +8,43 @@ class MergeServiceHealthcheckTests: XCTestCase {
 
     private let statusChecksTimeout = 30.minutes
 
+    private func makeState(status: MergeService.State.Status) -> MergeService.State {
+        return .init(
+            integrationLabel: .init(name: "Merge"),
```

Comment:
I would prefer that we move these definitions to a fixture, perhaps `LabelFixture`, and pass values defined in that fixture as default arguments to the `makeState` function. I think it is very likely that we will want customise our test cases with different values.

Reply:
@zzcgumn Suggestion applied in latest commits